registry/quilt_server/prod_config.py

```diff
@@ -10,7 +10,11 @@
 REGISTRY_URL = os.environ['REGISTRY_URL']
 CATALOG_HOST = os.environ['CATALOG_HOST']
 
-CATALOG_URL = 'https://%s' % CATALOG_HOST
+CATALOG_URL = os.environ.get('CATALOG_URL', 'https://%s' % CATALOG_HOST)
```

Comment:
I'm fine with `CATALOG_URL` - but we should then drop `CATALOG_HOST`. No point in having two of them